### ISSUE TITLE
Replace /payc?

### ISSUE THREAD

**Issue by TyJcksn** [Joannou1/GLServers#27]
Payc has gotten old, it's complicated to use and seems to be becoming outdated with all these new mobs that are being added to minecraft. I looked around on bukkit and found this plugin, it is capable with vault and has a GUI added to it.
 http://dev.bukkit.org/bukkit-plugins/spawner-changer-gui/

**Comment by Joannou1** [comment]
I love the idea, but the implementation would be really difficult.
Not sure how we could easily transfer over the existing spawners to the new plugin, as the locations for the spawner are stored in a file like this:

-19912|121|7935|3000: Joannou1

We have the plugin made a couple of years ago, so he had to use some tricky tricks to make it work.

**Comment by Kladdkaka** [comment]
We could give everyone 1 week to sell their spawners they dont want

**Comment by TyJcksn** [comment]
May have to be more than a week

**Comment by TyJcksn** [comment]
So this plugin won't affect any current spawners, unplaced? I'm slightly confused

**Comment by Joannou1** [comment]
Yeah, ideally I will find a way to switch plugins while maintaining the current spawners.
